llama_stack/distribution/build_container.sh

```diff
@@ -154,6 +154,12 @@ get_python_cmd() {
     fi
 }
 
+# Add other require item commands genearic to all containers
+add_to_container << EOF
```

Comment:
Is this needed for the external provider dir change?

Reply:
yeah, we need to RUN mkdir ~/.llama before doing the below where we COPY the system providers.d into the container's

this was happening below I just moved it up